packages/falso/src/lib/i18n/ru/first-name.ts

```diff
@@ -0,0 +1,35 @@
+import { fake } from '../../core/core';
+import { data } from './first-name.i18n.json';
+import { NameOptions } from '../../full-name';
+import { rand } from '../../rand';
+
+/**
+ * Generate a random first name.
+ *
+ * @category person
+ *
+ * @example
+ *
+ * randFirstName()
+ *
+ * @example
+ *
+ * randFirstName({ withAccents: true })
+ *
+ * @example
+ *
+ * randFirstName({ gender: 'female' }) // Emma
+ *
+ * @example
+ *
+ * randFirstName({ length: 10 })
+ *
+ */
+export function randFirstName<Options extends NameOptions = never>(
```

Comment:
We have discussed this approach in https://github.com/ngneat/falso/pull/212. The issue is that we don't have a single source of truth. It'll be better to find a way to use the same functions with different data.

Reply:
Can you please show an example of the usage?

Reply:
As you mentioned in #212 
```ts
import { randBird } from '@ngneat/falso';
import ruBird from '@ngneat/falso/i18n/ru/bird';

randBird({ locale: ruBird })
```
but for randFirstName